eagle-server/src/main/webapp/app/dev/public/js/ctrls/authCtrl.js

```diff
@@ -26,6 +26,14 @@
 		$scope.password = '';
 
 		$scope.login = function () {
+		if (!$scope.username || !$scope.password)
```

Comment:
@zombieJ could you please have a look about the changes?

Reply:
Hi @VSJTech,
Look good to me. And could you help to format the code?

Thanks